src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/caseassignment/AssignCaseToUserHandlerTest.java

```diff
@@ -280,6 +285,26 @@ void shouldAssignCaseToApplicantSolicitorOneAndRemoveCreator1v2DSUnregisteredRes
         }
     }
 
+    @Test
+    void handleEventsReturnsTheExpectedCallbackEvents() {
+        assertThat(assignCaseToUserHandler.handledEvents()).contains(ASSIGN_CASE_TO_APPLICANT_SOLICITOR1,
+                                                                     ASSIGN_CASE_TO_APPLICANT_SOLICITOR1_SPEC);
```

Comment:
Make it an exact match, so if people adds new events they are forced to look at the tests.